e2e/regression/testdata/nginx.yml

```diff
@@ -18,4 +18,30 @@ spec:
           image: ghcr.io/edgelesssys/nginx-unprivileged@sha256:1d5be2aa3c296bd589ddd3c9bf2f560919e31ac32bae799a15dd182b6fdb042b
           ports:
             - containerPort: 8080
+          volumeMounts:
+            - name: html
+              mountPath: /usr/share/nginx/html
+              readOnly: true
+      volumes:
+        - name: html
+          configMap:
+            name: nginx-html
       runtimeClassName: contrast-cc
+---
+apiVersion: v1
```

Comment:
Should we add a comment here stating that this is a regression test for a fix?

Reply:
Done